pmix-standard.tex

```diff
@@ -44,15 +44,38 @@
 % The following says letter size, but the style sheet may change the size
 \documentclass[10pt,letterpaper,twoside,makeidx,hidelinks]{scrreprt}
 
+%%%%%%%%%%%%%%%%%%%
+% Release Managers:
+% - Set is_unofficial_draft = false to remove the watermark
+% - Set VER to the correct version number
+% - Set VERDATE to the Month+Year of the release
+%%%%%%%%%%%%%%%%%%%
+\usepackage[us,24hr]{datetime}
+\usepackage{ifthen}
+\newboolean{is_unofficial_draft}
+\setboolean{is_unofficial_draft}{true}
```

Comment:
What is the intended procedure for unsetting/setting this boolean during a release?  Merge a PR that sets it to `false`, tag the release, then shortly after merge another PR that sets it back to `true`?

Reply:
Generally, yes - you would set it back to true _and_ up the revision number (adding an "a1" or something to indicate it is a working branch again) in a subsequent PR.

Reply:
I'd like the default value in the repo to be `true` so that if someone checks out the repo they will get the watermark. For the release tag point having it set to `false` in one commit seems reasonable to me.